packages/client/src/config/autofill.ts

```diff
@@ -1,51 +1,43 @@
-import { addQuotes, getRestrictionByTypeName } from "../utils";
+import {
+  AutofillData,
+  AutofillResolverMap,
+  Checker,
+  Question,
+  addQuotes,
```

Comment:
🔤

Reply:
it is.
I am using prettier and it sorts the imports this way.